inspirehep/modules/disambiguation/search.py

```diff
@@ -0,0 +1,365 @@
+
+    :param source:
+        A list of fields to be extracted from the Elasticsearch response.
+
+        Example:
+            source = ["collaboration", "author.full_name"]
+    """
+    elasticsearch_index = current_app.config.get('DISAMBIGUATION_RECORD_INDEX')
+    elasticsearch_type = current_app.config.get('DISAMBIGUATION_RECORD_TYPE')
+
+    return es.get_source(
+        index=elasticsearch_index,
+        id=str(record_id),
+        doc_type=elasticsearch_type,
+        _source=source)
+
+
+def _search(query):
+    """Make a call to the Elasticsearch instance.
+
+    Receives the dictionary as the parameter.
+    With the given paremeter, the Elasticsearch instance is being
+    queried.
+
+    :query:
+        The query for the Elasticsearch instance.
+
+        Example:
+            query = {"query": {"match_all": {}}}
+
+    :return:
+        The Elasticsearch instance response.
+    """
+    elasticsearch_index = current_app.config.get('DISAMBIGUATION_RECORD_INDEX')
+
+    return es.search(
+        index=elasticsearch_index,
+        body=query)['hits']['hits']
+
+
+def _scroll_search(query):
+    """Make a call to the Elasticsearch with a big response.
+
+    Receives the dictionary as the parameter.
+    With the given paremeter, the Elasticsearch instance is being
+    queried.
+
+    :query:
+        The query for the Elasticsearch instance.
+
+        Example:
+            query = {"query": {"match_all": {}}}
+
+    :return:
+        The Elasticsearch instance response.
+    """
+    elasticsearch_index = current_app.config.get('DISAMBIGUATION_RECORD_INDEX')
+    elasticsearch_type = current_app.config.get('DISAMBIGUATION_RECORD_TYPE')
+
+    result = []
+
+    for response in scan(
+            es, query=query,
+            index=elasticsearch_index,
+            doc_type=elasticsearch_type):
+        result.append(response)
+
+    return result
+
+
+def create_record(record_id):
+    """Create a record in the Beard readable format.
+
+    Receives a UUID of the given record as the parameter.
+    In order to run Beard, a list of dictionaries, representing records,
+    must be supplied. This method allows to create a dictionary
+    representing one record.
+
+    :param record_id:
+        A UUID of a given record.
+
+        Example:
+            record_id = "beb847d1-c7e0-48b2-9f48-a0d5d0586199"
+
+    :return:
+        A dictionary representing one record in the Beard readable format.
+
+        Example:
+            {'authors': [u'Hohm, Olaf', u'Wang, Yi-Nan'],
+             'publication_id': u'13c3cca8-b0bf-42f5-90d4-e3dfcced0511',
+             'year': u'2015'}
+    """
+    _source = [
+        "authors.full_name",
+        "earliest_date"
+    ]
+
+    response = _get_record(record_id, _source)
+
+    # Get the year of the record publication.
+    try:
+        earliest_date = response.get("earliest_date")
+        year = parse(earliest_date).strftime("%Y")
+    except:
+        year = False
```

Comment:
(Nit) eh, this is actually guaranteed in Python 3, but not necessarily true in Python 2:
```
In [1]: False = 5

In [2]: int(False)
Out[2]: 5
```